skimage/restoration/_denoise.py

```diff
@@ -275,7 +275,7 @@ def denoise_tv_bregman(image, weight, max_num_iter=100, eps=1e-3,
     weight : float
         Denoising weight. The smaller the `weight`, the more denoising (at
         the expense of less similarity to the `input`). The regularization
-        parameter `lambda` is chosen as `2 * weight`.
+        parameter `lambda` is chosen as `2 * weight`. Default weight is 5.0.
```

Comment:
```suggestion
        parameter `lambda` is chosen as `2 * weight`.
```

we don't typically list the defaults explicitly since they are apparent from the function signature.